tests/manage/z_cluster/nodes/test_nodes_maintenance.py

```diff
@@ -484,6 +489,16 @@ def test_pdb_check_simultaneous_node_drains(
         time.sleep(30)
         try:
             drain_nodes([node_B])
+            # After the drain check Mon failover in 10th, 15th and 20th min
+            timeout = [600, 300, 300]
+            for failover in timeout:
+                sample = TimeoutSampler(
+                    timeout=failover,
+                    sleep=10,
+                    func=helpers.check_mon_pods_eq_3,
+                )
+                if sample.wait_for_func_status(result=False):
+                    assert "Number of mon pods not equal to 3"
```

Comment:
Instead of the number 3, we can define a new var like `expected_mon_num=3`

Reply:
done